modular_skyrat/modules/alternative_job_titles/code/alt_job_titles.dm

```diff
@@ -122,29 +145,43 @@
 		"Deck Worker",
 		"Inventory Associate",
 		"Mailman",
+		"Mailwoman",
+		"Mail Carrier",
+		"Merchantman",
+		"Merchantwoman",
+		"Postman",
+		"Postwoman",
 		"Receiving Clerk",
 		"Union Associate",
 	)
 
 /datum/job/chaplain
 	alt_titles = list(
 		"Chaplain",
+		"Clockwork Priest",
+		"Cleric",
 		"High Priest",
+		"High Priestess",
 		"Imam",
 		"Magister",
 		"Monk",
+		"Nun",
 		"Oracle",
 		"Preacher",
 		"Priest",
+		"Priestess",
 		"Pontifex",
+		"Guru",
 		"Rabbi",
 		"Reverend",
-		"Cleric",
+		"Shrine Maiden",
+		"Shrine Guardian",
 	)
 
 /datum/job/chemist
 	alt_titles = list(
 		"Chemist",
+		"Alchemist",
```

Comment:
fix the L

Reply:
https://www.merriam-webster.com/dictionary/alchemist

Reply:
> https://www.merriam-webster.com/dictionary/alchemist

Ik what alchemist is, I didnt realize github was gaslighting me into thinking that is a 1.

Reply:
Oh lol.